auth0/v3/authentication/authorize_client.py

```diff
@@ -12,6 +15,25 @@ class AuthorizeClient(AuthenticationBase):
     def __init__(self, domain):
         self.domain = domain
 
+    def get_authorize_url(self, client_id, audience=None, state=None, redirect_uri=None,
```

Comment:
if you're going to create a method like this, it would make more sense to call it in `authorize` below instead of having the logic in two places.